Interoperability/Auth/Public/FirebaseAuthInterop.h

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
What's the purpose of this file?  I believe we only construct umbrella headers manually for legacy compatibility reasons.